app.js

```diff
@@ -3,7 +3,7 @@ import { User, Product } from './models';
 import DirWatcher from './dirWatcher';
 import Importer from './importer';
 
-console.log(config.name);
+// console.log(config.name);
```

Comment:
Please do not leave commented code in the branch you are going to merge with master. Keep code clean!

Reply:
Removed commented code.